src/video.c

```diff
@@ -1067,6 +1071,7 @@ static int vrx_print_pipeline(struct re_printf *pf, const struct vrx *vrx)
 int video_alloc(struct video **vp, struct list *streaml,
 		const struct stream_param *stream_prm,
 		const struct config *cfg,
+		const struct account *acc,
```

Comment:
Doxygen `@param acc  Account (optional)` is missing

Reply:
Thanks. Fixed.